community.md

```diff
@@ -144,6 +144,18 @@ If you send us an email with a code snippet, make sure that:
 3. you keep formatting when pasting code in order to keep the code readable
 4. there are enough import statements to avoid ambiguities
 
+## Slack
+
+You can join the [Apache Flink community on Slack.](https://join.slack.com/t/apache-flink/shared_invite/zt-19uavkkkw-qwMimi9gawIv9Fz9xFuqrw)
```

Comment:
The invite link is not the only way to join the workspace. Every existing member is free to invite new members, without needing approval from admins. We may want to mention and even suggest that. Hopefully that reduces the frequency we have to update the invite link.

Reply:
Good point, I've added a comment for that.